electron/app/chatBro/index.ts

```diff
@@ -1,24 +1,3 @@
-import * as _ from 'lodash';
-import * as sqlite3 from 'sqlite3';
-import { settleAll } from 'blend-promise-utils';
-import { createWordTable, getWordCount } from './queries/wordCount';
-import * as sqlite3Wrapper from './util/sqliteWrapper';
-import { ChatBro } from './definitions';
-
-const tableNames: string[] = _.values(ChatBro.Tables);
-
-// where all other table creation will live
-async function createAllTables(db: sqlite3.Database) {
-  return createWordTable(db);
-}
-
-async function dropAllExistingTables(db: sqlite3.Database) {
-  const dropTablePromises = tableNames.map(async (tableName) =>
-    sqlite3Wrapper.runP(db, `DROP TABLE IF EXISTS ${tableName}`)
-  );
-  return settleAll(dropTablePromises);
-}
-
-const tables = ChatBro.Tables;
-
-export { dropAllExistingTables, createAllTables, getWordCount, tables };
+export * from './db'
+export * from './tables'
+export * from './queries'
```

Comment:
basically moving out chat bro stuff internally to 3 different things, db, tables, and queries

db only creates/closes the db

tables creates/drops the tables

queries does the queries

Was thinking about it like a dependency graph of like

1. db
2. tables (db)
3. queries (db, tables)

Reply:
This is sick. Wanna add what you wrote here to the README?